src/services/dashboards/dashboard-customize/stores/widget-form.ts

```diff
@@ -0,0 +1,56 @@
+import { defineStore } from 'pinia';
+
+import type { DashboardLayoutWidgetInfo } from '@/services/dashboards/widgets/config';
+import { getWidgetConfig } from '@/services/dashboards/widgets/widget-helper';
+
+
+interface State {
+    widgetConfigId?: string;
+    name?: string;
+    isValid: boolean;
+    formData?: DashboardLayoutWidgetInfo;
```

Comment:
suggestion: I think widget form data can contain widget title and options. DashboardLayoutWidgetInfo type is too heavy for formData I think...🤔

Reply:
Oh, I set wrong type here! It's form data of JsonSchemaFor, so it's type is...just `object`.